internal/authorization/resource_manager.go

```diff
@@ -0,0 +1,79 @@
+package authorization
+
+import (
+	"context"
+)
+
+func (a *Authorizer) createTuple(ctx context.Context, resourceID string) error {
+	user, ok := ctx.Value(USER_CTX).(*User)
+	if !ok {
+		// Should we panic?
+		return nil
```

Comment:
return an error `fmt.Errorf("user not authenticated or requests context broken, cannot perform operation")`